ports/samd/boards/GENERIC_SAMD21J18/pins.csv

```diff
@@ -0,0 +1,10 @@
+# The lines contain pairs of Pin name and Pin number.
+# Pin names must be valid Python identifiers.
+# Pin numbers have the form Pxnn, with x being A, B, C or D.
+# Lines starting with # or empty lines are ignored.
+
```

Comment:
Is it worth exposing all standard GPIO, so they can be used if they are physically available?  Otherwise no pins can be used (or does samd always provide defaults?).

Reply:
All pins are available through Pin.cpu. So you can create a pin object e.g. as
Pin.cpu.PA01
Pin("PA01")
Pin(1)